src/index.ts

```diff
@@ -2,8 +2,9 @@ import * as AST from "./ast"
 import { RegExpParser } from "./parser"
 import { RegExpValidator } from "./validator"
 import { RegExpVisitor } from "./visitor"
+import { RegExpSyntaxError } from "./regexp-syntax-error"
```

Comment:
Since we're just doing an export and not using it in this file (like the other imports), we can simplify this to 

```suggestion
export { RegExpSyntaxError } from "./regexp-syntax-error"
```